### PR TITLE
New mob: Large Dragon, and some changes.

### PR DESCRIPTION
**Pull request by BlackMajor**
This PR includes the following.

A new, complex boss mob, simply called Large Dragon.
Read the .dm file for a summary at the top of its file of what it's about.

A new ruin, called Dragon Plateu for the Large Dragon to spawn on.
(Yo low key, the wildness is using a cap of 60 for both its northern and southern parts, coming up to 120 total points for ruin spawning. Y'all might want to consider bumping it up to 90, seeing as some ruins in that file cost 30.)

A proactive healbelly AI datum, which has promptly been applied to the Leopardmanders so they stop mauling people they should be healing.
Cleaned out the Leopardmander file a bit, too.

A nerfed variant of the Large Dragon which has been added to the maint pred spawn list.

A tiny adjustment to vore belly code to allow for vore organs to automatically transfer prey around over time, for the dragon's own vore organs. I did not bother adding this to the save files, cause why would you want auto transfer voregans in players? You can just transfer them when you want yourself.

Edit:
After some testing, a few tweaks to be made.
Updating this as folk play around with them.
Still need more combat testing because balance. Though so far most folk are just getting ate by them lmao.
Todo:
- [x] Finish the vore organ flavourtexts/descriptions
- [x] A few smol bug fixes (Healbelly not targetting unconscious people, layering above weather for some reason?  Layering putting mounted folk behind the sprite. Code hard lmao.)
- [x] The mob vore bellies auto move prey over time. I should either make that only work when not player controlled, or add the setting to player vore panels. Not sure which one I want to do.
- [x] Fix mob not warning players it's nomming them.
- [x] These dragons are bloody huge, add a client toggle to make the sprite smaller from your point of view so they can actually melee things when player controlled.
- [x] Low priority, but see if I can fix the speech bubble/radial/runechat/grab offsets
- [x] Fix an abnormal bug where it's AI holder deletes when going from hostile -> friendly -> hostile
- [x] Add a toggle for turning your firebreath on/off
- [x] Add a toggle for turning your light on/off
- [x] Improve the radial icon selection with previews
- [x] Add a way for player controlled dragons to self heal in a balanced way

### CODE REVIEW

File: code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm [hunk 1]
```diff
@@ -0,0 +1,864 @@
...
+	vore_sound = "Tauric Swallow"
+	escapable = 0
+	//desc = "The dragon's cavernous stomach hungrily pries your figure in through its sphincter, sealing taught behind you. Its meal is hastily subject to the slick, squelching confines of its digestive organ, that hanging gut eagre to churn and ripple over your proportions the moment you've arrived. Caustic waves of slop and chyme wash over you, that acidic tingle setting in upon your flesh. The entire chamber groans and echoes with each of the dragon's prideful steps. Satiated with its most recent meal, slowly being reduced into a nutritious paste to further power its hunt. Your fate definitively sealed within this tomb of swelteringly hot flesh."
```
Comment (Shadow-Quill) [inline code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm:491]: _squint_

File: code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm [hunk 2]
```diff
@@ -0,0 +1,864 @@
...
+Unless you're a tesh or something I guess. Speed.
+
+As a flex, this mob has six vore organs.
+It will swap between two variants (heal and digest) depending on if it's friendly or not.
+It will eat prey with its maw, which has code in place to automatically push prey deeper over time.
+The mob will only get increased "fullness" for prey who have made it to its stomach. Y'know. So the fat sprites make sense.
+
+This mob can be made friendly by supplying it with a gold coin or a gold ingot. I'd make it every gold item, but then you could just raid its hoard for an easy tame. And that's lame.
+
+While friendly, it will actively search for players with >95% health and attempt to vore them into its heal bellies.
```
Comment (Shadow-Quill) [inline code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm:38]: Assuming this is meant to be `<95%`

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm (hunk 2 above)
Comment (BlackMajor) [reply code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm:38]: Yes

File: code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm (hunk 1 above)
Comment (BlackMajor) [reply code/modules/mob/living/simple_mob/subtypes/vore/bigdragon_ch.dm:491]: Still fluffing out the vore organ descriptions and such.

**Comment by Darlantanis** [comment]
You're missing a comment in event_vr.dm.

**Pull request merged**